concept_mapper_experiment/01_create_concept_to_id_mapper.py

```diff
@@ -0,0 +1,231 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:biovectors]
+#     language: python
+#     name: conda-env-biovectors-py
+# ---
+
+# # Extract all concept IDs for Pubtator Central
+
+# +
+import gzip
+import os
+from pathlib import Path
+import re
+
+import pandas as pd
+import plydata as ply
+import tqdm
+# -
+
+Path("output/intermediate_files").mkdir(exist_ok=True, parents=True)
+
+# # Extract MESH IDs
+
+mesh_chemical_url = (
+    "https://nlmpubs.nlm.nih.gov/projects/mesh/MESH_FILES/asciimesh/c2022.bin"
+)
+mesh_disease_url = (
+    "https://nlmpubs.nlm.nih.gov/projects/mesh/MESH_FILES/asciimesh/d2022.bin"
+)
+
+# +
+chemical_file = f"output/intermediate_files/{Path(mesh_chemical_url).name}"
+if not Path(chemical_file).exists():
+    os.system(f"wget {mesh_chemical_url} -O {chemical_file}")
+
+disease_file = f"output/intermediate_files/{Path(mesh_disease_url).name}"
+if not Path(disease_file).exists():
+    os.system(f"wget {mesh_disease_url} -O {disease_file}")
+# -
+
+mesh_map_file = Path("output/mesh_headings_id_mapper.tsv")
+if not mesh_map_file.exists():
+    with open(disease_file, "r") as disease_infile, open(
+        chemical_file, "r"
+    ) as chem_infile:
```

Comment:
Took me a second to realize you were opening two files here - maybe something like this would be clearer:
```suggestion
    with (
        open(disease_file, "r") as disease_infile,
        open(chemical_file, "r") as chem_infile
    ):
```